reactor-netty-core/src/test/java/reactor/netty/tcp/TcpClientTests.java

```diff
@@ -371,9 +365,8 @@ private void connectionWillRetryConnectionAttemptWhenItFails(TcpClient client)
 		                                    })))
 		         .subscribe(System.out::println);
 
-		assertTrue(latch.await(5, TimeUnit.SECONDS));
-		assertEquals("latch was counted down:" + latch.getCount(), 0, latch.getCount());
-		assertThat("totalDelay was >1.6s", totalDelay.get(), greaterThanOrEqualTo(1600L));
+		assertThat(latch.await(5, TimeUnit.SECONDS)).isTrue();
```

Comment:
the message failure for such an assertion typically says something like "expected <false> to be <true>", so adding an `as` description is helpful. I see you've done that in other places 👍 

```suggestion
		assertThat(latch.await(5, TimeUnit.SECONDS)).as("latch await").isTrue();
```